packages/mds-cache/redis/index.ts

```diff
@@ -57,7 +57,16 @@ export const RedisCache = () => {
 
     set: async (key: KeyType, val: ValueType) => safelyExec(theClient => theClient.set(key, val)),
 
-    expireat: async (key: KeyType, time: Timestamp) => safelyExec(theClient => theClient.expireat(key, time)),
+    /**
+     * Expires at Unix time in seconds
+     */
+    expireat: async (key: KeyType, timeInSeconds: Timestamp) =>
+      safelyExec(theClient => theClient.expireat(key, timeInSeconds)),
```

Comment:
`Timestamp` is a misleading type-alias to use here, the `Timestamp` type specifically is meant to represent MS. Should make a type alias `TimestampSeconds`, even though they're referencing the same type (`number`), or just make this a number I think.

Also, would probably make sense to make this take an _object_ for the options so that the user of this function *needs* to know that this is labeled as seconds. E.g. usage would be like `expireAt({ key: 'foo', timeInSeconds: 12345 })` instead of `expireAt('foo', 12345)`. Different API than ioredis exposes, but that's one of the benefits of being able to wrap these functions ourselves.

Reply:
fixed!